rest-client-base/src/main/java/com/wultra/core/rest/client/base/RestClientConfiguration.java

```diff
@@ -30,12 +29,6 @@
  */
 public class RestClientConfiguration {
 
-    /**
-     * Constructor of REST client configuration.
-     */
-    public RestClientConfiguration() {
```

Comment:
Useless no-arg constructor, same as the default one.

Reply:
Please do not remove the no-arg constructor. Jackson and similar frameworks require a no-arg constructor to function properly. We do not serialize the configuration into JSON right now but we might one day.

Reply:
I take it back... there is no other constructor. Ok, we can remove it.

Reply:
There is no other constructor, so default makes the same result.

Reply:
I see, ok, let's remove it then.